modules/ROOT/pages/migration-connectors-email.adoc

```diff
@@ -129,13 +129,13 @@ accessible through DataWeave. For instance, to reference an attachment called
 </flow>
 ----
 
-On the Mule 3 transport, the properties of the outgoing e-mail (subject, 
-addresses, etc) could be configured via outbound properties set in the event 
-before the execution of the outbound smtp endpoint. In that case, the outbound 
-properties would take precedence over the values configured in the endpoint or 
-connector themselves.  
+In the Mule 3 transport, the properties of the outgoing email (subject, 
+addresses, and so on) could be configured through outbound properties set in the event 
+before the execution of the outbound SMTP endpoint. In that case, the outbound 
+properties took precedence over the values configured in the endpoint or 
+connectors.  
 
-Unlike Mule 3, attachments to be sent with an email have to be configured 
+Unlike Mule 3, attachments to be sent with an email must be configured 
```

Comment:
attachments to be sent with an email in Mule 4? Not clear what the contrast is. Also, "are configured" is fine. Everything we tell them to do must be done.